### PR TITLE
Adding some changes to the object lock documentation

### PR DESCRIPTION
**Pull request by IBM-diksha**
Adding some naming convention changes to the objectlock documentation.

### CODE REVIEW

File: website/docs/d/cos_bucket.html.markdown [hunk 1]
```diff
@@ -96,9 +96,9 @@ resource "ibm_cos_bucket" "smart-us-south" {
 
 ```
 
-# ibm_cos_objectlock_configuration
+# ibm_cos_object_lock_configuration
 
-Retrive an IBM Cloud Object Storage bucket with objectlock enabled and objectlock configuration set on the bucket.It allows objectlock configuration to be updated or deleted.But it does not allow to disable objectlock as objectloock cannot be disabled once enabled on a bucket.
+Retrive an IBM Cloud Object Storage bucket with Object Lock enabled and Object Lock configuration set on the bucket.It allows Object Lock configuration to be updated or deleted.But it does not allow to disable Object Lock as objectloock cannot be disabled once enabled on a bucket.
```
Comment (IBMalok) [inline website/docs/d/cos_bucket.html.markdown:101]: space after `set on the bucket. It`

Comment (IBMalok) [reply website/docs/d/cos_bucket.html.markdown:101]: after `updated or deleted.But `

Comment (IBMalok) [reply website/docs/d/cos_bucket.html.markdown:101]: two times oo `objectloock`

File: website/docs/r/cos_bucket.html.markdown [hunk 2]
```diff
@@ -398,9 +398,9 @@ resource "ibm_cos_bucket" "cos_bucket_onerate" {
 
 
 ```
-# ibm_cos_objectlock_configuration
+# ibm_cos_object_lock_configuration
 
-COS objectlock feature enables user to store the object in a bucket with an extra layer of protection against object changes and deletion.Object Lock can help prevent objects from being deleted or overwritten for a fixed amount of time or indefinitely by setting up retention period and legalhold for an object.
+COS Object Lock feature enables user to store the object in a bucket with an extra layer of protection against object changes and deletion.Object Lock can help prevent objects from being deleted or overwritten for a fixed amount of time or indefinitely by setting up retention period and legal hold for an object.
```
Comment (IBMalok) [inline website/docs/r/cos_bucket.html.markdown:403]: space after dot `hanges and deletion.Object `

File: website/docs/r/cos_bucket_object_lock_configuration.html.markdown [hunk 3]
```diff
@@ -59,13 +59,13 @@ resource ibm_cos_bucket_object_lock_configuration "objectlock" {
 }
 
 ```
-# Enabling Objectlockconfguration on an existing bucket
-To enable objectlock configuration on an existing bucket , create a COS bucket with object versioning enabled and pass the crn of the COS bucket and location of the bucket to `ibm_cos_bucket_object_lock_configuration.bucket_crn` and `ibm_cos_bucket_object_lock_configuration.bucket_location` as shown in the example.
+# Enabling Object Lock configuration on an existing bucket
+To enable  Object Lock configuration on an existing bucket , create a COS bucket with object versioning enabled and pass the crn of the COS bucket and location of the bucket to `ibm_cos_bucket_object_lock_configuration.bucket_crn` and `ibm_cos_bucket_object_lock_configuration.bucket_location` as shown in the example.
```
Comment (IBMalok) [inline website/docs/r/cos_bucket_object_lock_configuration.html.markdown:63]: extra space after the bucket `To enable  Object Lock configuration on an existing bucket ,`

**Review by IBMalok — APPROVED** [review]

**Review by hkantare — APPROVED** [review]

**Pull request merged**